mqutils/utils.py

```diff
@@ -35,8 +43,3 @@ def pixel_distance(stars):
             distances[i][i + 1:] = dis
     logger.info('The End!')
     return distances
-
-""" Alternatively, you can use the following suggestion for calculating the Eucliudean Distance between the pairs of coordinates. 
```

Comment:
@Shalmalee15 
This comment is related to the function in the file, so maybe it is better to move it inside the function. What do you think about the way that I moved it?